ui-tests/src/test/java/org/openmrs/reference/page/AttachmentsPage.java

```diff
@@ -0,0 +1,54 @@
+package org.openmrs.reference.page;
+
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
```

Comment:
Go on to write a test class for these changes just to ensure that the elements in these file work as expected

Reply:
> Go on to write a test class for these changes just to ensure that the elements in these file work as expected

As mentioned in the ticket description, this is a resource to selenium automation tests in qaframework module. Have written the test class already in qaframework module. Here is the [pr](https://github.com/openmrs/openmrs-contrib-qaframework/pull/16) to this test.

Reply:
Am gona try to get some minutes and test things locally